acsf-convert/files/hooks/common/post-db-copy/db-scrub.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+#
+# db-copy Cloud hook: db-scrub
+#
+# Scrub important information from a Drupal database.
+#
+# Usage: db-scrub.sh site target-env db-name source-env
+
+set -ev
+
+site="$1"
+target_env="$2"
+db_name="$3"
+source_env="$4"
+
+# Prep for BLT commands.
+repo_root="/var/www/html/$site.$target_env"
+export PATH=$repo_root/vendor/bin:$PATH
+cd $repo_root
+
+blt artifact:ac-hooks:db-scrub $site $target_env $db_name $source_env -D drush.ansi=false
+
+set +v
```

Comment:
The approach I usually take is, did we have it, or similar, there before? Additionally, is it called from the factory-hooks?

Reply:
This file was generated from the acsf:init command. Added it so we know exactly what it adds. If we want to remove it we can as it will run the scrub hooks anytime a DB is copied on ACSF. 

Seems this could be problematic as its in common and could run if you ever copied a DB up to prod... 

Do we want to comment out the command it doesnt run on ACSF?